bundle/manifests/3scale-operator.clusterserviceversion.yaml

```diff
@@ -1011,9 +1038,9 @@ spec:
         serviceAccountName: 3scale-operator
     strategy: deployment
   installModes:
-  - supported: true
+  - supported: false
     type: OwnNamespace
-  - supported: true
+  - supported: false
```

Comment:
When adding conversion webhook, it looks must only `AllNamespace` install mode should be enabled, otherwise the bundle will fail validation
```
➜  3scale-operator git:(THREESCALE-8002) ✗ make bundle-validate
/Users/chfan/dev/3scale-operator/bin/operator-sdk bundle validate ./bundle
INFO[0000] Found annotations file                        bundle-dir=bundle container-tool=docker
INFO[0000] Could not find optional dependencies file     bundle-dir=bundle container-tool=docker
ERRO[0000] Error: Value : (3scale-operator.v0.0.1) only AllNamespaces InstallModeType is supported when conversionCRDs is present 
make: *** [bundle-validate] Error 1
```